src/components/Detail/Detail.css

```diff
@@ -0,0 +1,19 @@
+.detail {
+  align-self: start;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.title {
+  color: darkorange !important;
```

Comment:
use ` rgb(233, 124, 52) `

Reply:
Yeah I think that color looks better than darkorange, but I'm just using that color because It's easier to remember. We should create a base css file, so we can reuse that styles.